Tests/SonarQube.Common.UnitTests/VerbosityCalculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestUtilities;
+
+namespace SonarQube.Common.UnitTests
+{
+    [TestClass]
+    public class VerbosityCalculatorTests
```

Comment:
There should be tests for the properties method too (test as close to the code as possible).